js/types.ts

```diff
@@ -588,6 +592,9 @@ export interface IMusicMetadataBrowserApi {
 export interface Extras {
   requireJSZip: () => Promise<never>;
   requireMusicMetadata: () => Promise<IMusicMetadataBrowserApi>;
+  handleTrackDropEvent: (
+    e: React.DragEvent<HTMLDivElement>
+  ) => Promise<Track[] | null>;
```

Comment:
Why don't we make it "awaitable" meaning, it can return: `Track[] | null | Promise<Track> | Promise<null>`